tasks/main.yml

```diff
@@ -10,11 +10,32 @@
     enabled: true
     state: 'started'
 
+- name: Check if firewalld is installed
+  command:
+  args:
+    cmd: 'rpm -q firewalld'
+    warn: false
+  register: check_for_firewalld
+  failed_when: false
+  changed_when: false
+  check_mode: false
+
```

Comment:
Is it necessary to check if firewalld package is installed if we later check the service state? (check my next comment for suggestion)

Reply:
See new commit b067df7